clang/www/cxx_dr_status.html

```diff
@@ -14478,7 +14478,7 @@ <h2 id="cxxdr">C++ defect report implementation status</h2>
     <td><a href="https://cplusplus.github.io/CWG/issues/2445.html">2445</a></td>
     <td>C++20</td>
     <td>Partial ordering with rewritten candidates</td>
-    <td class="unknown" align="center">Unknown</td>
+    <td class="unreleased" align="center">Clang 18</td>
```

Comment:
You shouldn't edit this file by hand. Instead, you should place a test in `/clang/test/CXX/drs/24xx.cpp` with a `// dr2445: 18` status, then run `/clang/www/make_cxx_dr_status` script. Follow the example of other tests in `24xx.cpp` while doing that.

Reply:
I'm on it. While we are at here: should it be `18` or `19` ? As I can see 18 is almost out.

Reply:
`19`, unless you're planning to cherry-pick this into 18 branch (I don't advise to do that this late into pre-release phase).

Reply:
done